filter_list/list_filter_example/filter_list.py

```diff
@@ -0,0 +1,9 @@
+class lists_class_filter:
```

Comment:
class name should follow camel case. I have already given you the google python coding standard. Please follow that.